src/lib/inbox/inbox.ts

```diff
@@ -276,4 +306,74 @@ export class InboxTools {
       overrides || {}
     )
   }
+
+  /**
+   * Send l2 signed tx using delayed inox, which won't alias the sender's adddress
+   * It will auto-confirm on l2, if it doesn't confirm in 24 hours, you can force
```

Comment:
We usually use the word `include` instead of auto-confirm. Maybe:

```
It will be automatically included by the sequencer on l2, if it isn't included within 24 hours, you can force include it
```